installscripts/jazz-terraform-unix-noinstances/chefProvisioner.tf

```diff
@@ -101,14 +101,18 @@ resource "null_resource" "configureJenkinsInstance" {
       "sudo rm -rf -f ${var.chefDestDir}"
     ]
   }
+  provisioner "local-exec" {
+    command = "bash ${var.configureJenkinsCE_cmd} ${lookup(var.jenkinsservermap, "jenkins_elb")} ${var.cognito_pool_username} '/var/lib/jenkins' ${lookup(var.scmmap, "scm_elb")} ${lookup(var.scmmap, "scm_username")} ${lookup(var.scmmap, "scm_passwd")} ${lookup(var.scmmap, "scm_privatetoken")} ${lookup(var.jenkinsservermap, "jenkinspasswd")} ${lookup(var.scmmap, "scm_type")} ${lookup(var.codeqmap, "sonar_username")} ${lookup(var.codeqmap, "sonar_passwd")} ${var.aws_access_key} ${var.aws_secret_key} ${var.cognito_pool_password} ${lookup(var.jenkinsservermap, "jenkinsuser")}"
+  }
 }
 
 resource "null_resource" "configureJenkinsDocker" {
   count = "${var.dockerizedJenkins}"
   depends_on = ["null_resource.preJenkinsConfiguration", "aws_elasticsearch_domain.elasticsearch_domain"]
-  // Chef Process inside docker
+
+  #Jenkins Cli process
   provisioner "local-exec" {
-    command = "bash ${var.launchJenkinsCE_cmd}"
+    command = "bash ${var.configureJenkinsCE_cmd} ${lookup(var.jenkinsservermap, "jenkins_elb")} ${var.cognito_pool_username} '/var/jenkins_home' ${lookup(var.scmmap, "scm_elb")} ${lookup(var.scmmap, "scm_username")} ${lookup(var.scmmap, "scm_passwd")} ${lookup(var.scmmap, "scm_privatetoken")} ${lookup(var.jenkinsservermap, "jenkinspasswd")} ${lookup(var.scmmap, "scm_type")} ${lookup(var.codeqmap, "sonar_username")} ${lookup(var.codeqmap, "sonar_passwd")} ${var.aws_access_key} ${var.aws_secret_key} ${var.cognito_pool_password} ${lookup(var.jenkinsservermap, "jenkinsuser")}"
```

Comment:
Both of the `configureJenkinsCE_cmd` invocations are exactly identical now. Which is a good thing, just make them one Terraform resource that always gets invoked at the end of both scenario, rather than copypasting the same invocation in 2 different spots.

Reply:
Done